src/libs/PolicyUtils.ts

```diff
@@ -318,12 +319,8 @@ function getPolicyIDFromNavigationState() {
     return getPolicyIDFromState(navigationRef.getRootState() as State<RootStackParamList>);
 }
 
-function getAdminEmailList(policy: Policy | null): Array<{email: string}> {
-    return Object.values(policy?.employeeList ?? {})
-        .filter((employee) => employee.email && employee.role === CONST.POLICY.ROLE.ADMIN)
-        .map((employee) => ({
-            email: employee.email ?? '',
-        }));
+function getAdminEmailList(policy: Policy | null): PolicyEmployee[] {
```

Comment:
```suggestion
function getAdminEmployees(policy: OnyxEntry<Policy>): PolicyEmployee[] {
```
Since this function returns the employees and not just their emails